src/utils.js

```diff
@@ -5,6 +5,31 @@ var utils = {},
     mailer = require("./mailer"),
     config = require('./config');
 
+utils.formatPodInfo = function(pod) {
```

Comment:
I'd prefer any new code always use the term `node` over `pod` - as this site is not only related to diaspora. Mind going through the whole PR and fixing those?